website/intro/getting-started/install.html.markdown

```diff
@@ -57,6 +57,17 @@ If you get an error that `terraform` could not be found, your `PATH` environment
 variable was not set up properly. Please go back and ensure that your `PATH`
 variable contains the directory where Terraform was installed.
 
+## Installing Terraform Providers
+
+Terraform uses a plugin based architecture to support the numerous infrastructure 
```

Comment:
Terraform uses a plugin-based architecture to interact with the numerous infrastructure and service providers available. Here is the [full list of Terraform Providers](/docs/providers/index.html). Importantly, the core Terraform binary installed above will automatically download and install Provider plugins for you; they do not need to be installed separately. Using Providers will be explained in more detail in the next section.